app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt

```diff
@@ -119,6 +109,85 @@ class AdyenPaymentFragment : BasePageViewFragment(), AdyenPaymentView {
+
+  // adyen_credit_card_layout.xml and adyen_credit_card_pre_selected.xml
+  private val fragment_credit_card_authorization_progress_bar get() = _bindingCreditCardPreSelected?.fragmentCreditCardAuthorizationProgressBar ?: _bindingCreditCardLayout?.fragmentCreditCardAuthorizationProgressBar!!
+  private val fiat_price_skeleton get() = _bindingCreditCardPreSelected?.paymentMethodsHeader?.fiatPriceSkeleton?.root ?: _bindingCreditCardLayout?.paymentMethodsHeader?.fiatPriceSkeleton?.root!!
+  private val appc_price_skeleton get() = _bindingCreditCardPreSelected?.paymentMethodsHeader?.appcPriceSkeleton?.root ?: _bindingCreditCardLayout?.paymentMethodsHeader?.appcPriceSkeleton?.root!!
+  private val iab_activity_transaction_completed get() = _bindingCreditCardPreSelected?.fragmentIabTransactionCompleted?.iabActivityTransactionCompleted ?: _bindingCreditCardLayout?.fragmentIabTransactionCompleted?.iabActivityTransactionCompleted!!
+
+  // adyen_credit_card_layout.xml
+  private val main_view get() = _bindingCreditCardLayout?.mainView
+  private val credit_card_info get() = _bindingCreditCardLayout?.creditCardInfo
+  private val line_separator get() = _bindingCreditCardLayout?.lineSeparator
+  private val cc_info_view get() = _bindingCreditCardLayout?.ccInfoView
+  private val change_card_button get() = _bindingCreditCardLayout?.changeCardButton
+  private val bonus_msg get() = _bindingCreditCardLayout?.bonusMsg
+  private val bonus_layout get() = _bindingCreditCardLayout?.bonusLayout?.root
+  private val adyen_card_form get() = _bindingCreditCardLayout?.adyenCardForm?.root
+  private val fragment_adyen_error get() = _bindingCreditCardLayout?.fragmentAdyenError?.root
+  private val error_buttons get() = _bindingCreditCardLayout?.errorButtons?.root
+
+  // adyen_credit_card_pre_selected.xml
+  private val main_view_pre_selected get() = _bindingCreditCardPreSelected?.mainViewPreSelected
+  private val payment_methods get() = _bindingCreditCardPreSelected?.paymentMethods
+  private val payment_scroll get() = _bindingCreditCardPreSelected?.paymentScroll
+  private val change_card_button_pre_selected get() = _bindingCreditCardPreSelected?.changeCardButtonPreSelected
+  private val more_payment_methods get() = _bindingCreditCardPreSelected?.morePaymentMethods!!
+  private val bonus_msg_pre_selected get() = _bindingCreditCardPreSelected?.bonusMsgPreSelected
+  private val bottom_separator_buttons get() = _bindingCreditCardPreSelected?.bottomSeparatorButtons
+  private val bonus_layout_pre_selected get() = _bindingCreditCardPreSelected?.bonusLayoutPreSelected?.root
+  private val layout_pre_selected get() = _bindingCreditCardPreSelected?.layoutPreSelected?.root
+  private val fragment_adyen_error_pre_selected get() = _bindingCreditCardPreSelected?.fragmentAdyenErrorPreSelected?.root
+  private val dialog_buy_buttons_error get() = _bindingCreditCardPreSelected?.dialogBuyButtonsError?.root
```

Comment:
why did you create these values ?

Reply:
I created these values because in the future the merge will be easier to do. If I would change every line where the variable is to "_bindingCreditCardPreSelected?.dialogBuyButtonsError?.root", in the future, we would have much more conflicts when merging. I tried to avoid changing the code as much as I could. After merging, we can just remove the properties that are not used.